documentation/tools/dataprotector/methods/fetchgrantedaccess.md

```diff
@@ -55,6 +71,35 @@ Filter the granted access by an authorized user address.
 
 <pre class="language-javascript"><code class="lang-javascript">const listGrantedAccess = await dataProtector.fetchGrantedAccess({
     protectedData: '0xA0Cf798816D4b9b9866b5330EEa46a18382f251e',
-<strong>    authorizedUser: '0x4a3B7c1eF78fA81bDf1aAa314D09a97578b0CA37'
-</strong>})
+    authorizedApp: '0x1f9840a85d5af5bf1d1762f925bdaddc4201f984',
+<strong>    authorizedUser: '0x4a3B7c1eF78fA81bDf1aAa314D09a97578b0CA37',</strong>
+    page: 1,
+    pageSize:100
+})
+</code></pre>
+
+### page (optional)
+
+Specifies the results page to return. If using this field you may also specify a `pageSize`. The default for this is `0` but if using paging, the functionality begins counting from index `1`.
```

Comment:
```suggestion
Specifies the results page to return. If using this field you may also specify a `pageSize`. The default for this is `0` but if using paging, the functionality begins counting from the value specify for this parameter.
```

Reply:
it's not always 1

Reply:
What I meant here is it's 1-indexed rather than 0-indexed. I'll try to rephrase this to be more clear.